.github/workflows/build-envoy-images-release.yaml

```diff
@@ -2,7 +2,7 @@ name: Refresh test & build cache & build latest
 on:
   push:
     branches:
-      - master
+      - v1.23
```

Comment:
Looks like this change will stop the release builds off of the master branch?

Reply:
So why not add the release branches and keep the master branch?

Reply:
And maybe keep `ENVOY_MINOR_RELEASE` as `master` for the master branch, as it will happen that master and a release branch will have the same Envoy minor version at some point in near future?

Reply:
> Looks like this change will stop the release builds off of the master branch?

ah this PR is merging to branch v1.23, which will contain the source code for proxy 1.23.x release. It's similar to what we have with cilium/cilium repo. 

The master branch is mantained separately, required changes are done #148.

Reply:
Right, missed that nuance.